clients/cobol-lsp-vscode-extension/src/__tests__/services/copybook/download/DownloadStrategyResolverTest.spec.ts

```diff
@@ -185,5 +191,34 @@ describe("tests download resolver", () => {
         "uss",
       );
     });
+    it("checks the order of copybook resolution - USS and DSN is not called when E4E resolves)", async () => {
+      (downloader as any).e4eDownloader.downloadCopybookE4E = jest
+        .fn()
+        .mockReturnValue(true);
+      (downloader as any).dsnDownloader.downloadCopybook = jest
+        .fn()
+        .mockReturnValue(false);
+      (downloader as any).ussDownloader.downloadCopybook = jest
+        .fn()
+        .mockReturnValue(false);
+      (E4ECopybookService as any).getE4EClient = jest.fn().mockReturnValue({});
```

Comment:
This condition indicates that it should not resolve - the copybook is not there.

Reply:
Not really, it indicates that It is not relevant to e4e